scipy/signal/wavelets.py

```diff
@@ -330,6 +412,12 @@ def cwt(data, wavelet, widths):
         satisfies these requirements.
     widths : (M,) sequence
         Widths to use for transform.
+    dtype : data-type, optional
+        The desired data type of output. Default is numpy.float64.
+        .. versionadded:: 0.18.2
```

Comment:
0.18.2 --> 1.1.0

Reply:
I'll have to bump this again as I'm rebasing / checking locally